framework_tvb/tvb/adapters/datatypes/db/__init__.py

```diff
@@ -49,7 +49,7 @@
     "SurfaceIndex": SurfaceRemover,
     "LocalConnectivityIndex": SurfaceRemover,
     "ConnectivityIndex": ConnectivityRemover,
-    "RegionVolumeMappingIndex": ConnectivityRemover,
+    "RegionVolumeMappingIndex": VolumeRemover,
     "RegionMappingIndex": RegionMappingRemover,
     "TimeSeriesRegionIndex": RegionVolumeMappingRemover,
     "SensorsIndex": SensorRemover,
```

Comment:
These removers are not fully correct. We should have one remover dedicated per each FK type.
I will refactor this immediately in this branch